### PR TITLE
Gseifu/update vcpkg

### PR DESCRIPTION
**Pull request by GelilaSeifu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ For additional functionality, see the public headers, located in `include/hexl`
 
 ## Building Intel HEXL
 
-Intel HEXL can be built in several ways. Intel HEXL has been uploaded to the [Microsoft vcpkg](https://github.com/microsoft/vcpkg) C++ package manager, which supports Linux, macOS, and Windows builds. See the vcpkg repository for instructions to build Intel HEXL with vcpkg, e.g. run `vcpkg install hexl`. There may be some delay in uploading porting the latest release to vcpkg, so please build from source to use the latest change in Intel HEXL.
+Intel HEXL can be built in several ways. Intel HEXL has been uploaded to the [Microsoft vcpkg](https://github.com/microsoft/vcpkg) C++ package manager, which supports Linux, macOS, and Windows builds. See the vcpkg repository for instructions to build Intel HEXL with vcpkg, e.g. run `vcpkg install hexl`. There may be some delay in uploading porting the latest release to vcpkg. Intel HEXL provides port files to build the latest version with vcpkg. Run `vcpkg install hexl --overlay-ports=/path/to/hexl/port/hexl --head`. For detailed explanation, see instruction for building vcpkg port using overlays provided by vcpkg.
```
Comment (jlhcrawford) [inline README.md:44]:
```suggestion
Intel HEXL can be built in several ways. Intel HEXL has been uploaded to the [Microsoft vcpkg](https://github.com/microsoft/vcpkg) C++ package manager, which supports Linux, macOS, and Windows builds. See the vcpkg repository for instructions to build Intel HEXL with vcpkg, e.g. run `vcpkg install hexl`. There may be some delay in uploading porting the latest release to vcpkg. Intel HEXL provides port files to build the latest version with vcpkg. Run `vcpkg install hexl --overlay-ports=/path/to/hexl/port/hexl --head`. For a detailed explanation, see instructions for building vcpkg port using overlays provided by vcpkg.
```

Comment (jlhcrawford) [reply README.md:44]: Typo? - `uploading porting`
Minor comment - is there a link to the "detailed explanation" or is it easy to find from the `Microsoft vcpkg` link?

**Review by jlhcrawford — APPROVED** [review]
LGTM

**Review by fboemer — APPROVED** [review]

**Comment by GelilaSeifu** [comment]
Changing to a new branch.

**Pull request closed without merging**